package.json

```diff
@@ -24,8 +24,9 @@
     "papaparse": "^5.4.1"
   },
   "devDependencies": {
-    "@playwright/test": "1.37.0",
+    "@playwright/test": "1.38.0",
     "@prettier/plugin-xml": "^3.1.0",
+    "@types/node": "^20.6.0",
```

Comment:
I needed to add node types after upgrading Playwright to 1.38.0

Reply:
In pom.xml we configure to download Node 18 at build-time: https://github.com/Sage-Bionetworks/SynapseWebClient/blob/73d467eea7cecfba65b3a65a057235919da13dc1/pom.xml#L361

Recommend using 18 (LTS) if we can. If not, change pom.xml to use v20

Reply:
Oh good call! I'll switch this to use 18.6